### PR TITLE
Go APIVIEW: Adding README

### PR DESCRIPTION
**Pull request by catalinaperalta**
This PR adds a README for the Go APIView tool and improves the command's usage description.

### CODE REVIEW

File: src/go/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Getting started
+
+### Prerequisites
+
+- Go 1.13 or above.
+
+### Generate the tool
+
+To create the command (this will output an executable), run in the `/go` directory:
```
Comment (jhendrixMSFT) [inline src/go/README.md:11]: I would reword this slightly, something like.

"To build the tool, execute the following command from the `./src/go` directory."

File: src/go/cmd/root.go [hunk 2]
```diff
@@ -12,12 +12,11 @@ import (
...
 // rootCmd represents the base command when called without any subcommands
 var rootCmd = &cobra.Command{
-	Use:   "apiview",
+	Use:   "apiview <packageDir> <outputDir>",
 	Short: "apiview will generate the JSON token output for APIView",
-	Long: `apiview is a tool that can be used to generate the tokenized 
-		output needed by the APIView tool to show a view of publicly 
-		exposed portions of an API.
-		To generate the output run: apiview <packageDir> <outputDir>`,
+	Long: `The apiview command can be used to generate the tokenized
```
Comment (jhendrixMSFT) [inline src/go/cmd/root.go:17]: I would include the name of the file in the help someplace.  Also, if running it will overwrite any preexisting output file it would be good to call that out too.

File: src/go/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+```
+go build
+```
+
+NOTE: To access the command anywhere you can add it to your $PATH variable. 
+
+### Run the tool
+
+Output a JSON file with the tokenized output for the SDK by running:
```
Comment (jhendrixMSFT) [inline src/go/README.md:20]: Run the following command to generate the <filename> file containing the tokenized output for the SDK.

File: src/go/cmd/root.go (hunk 2 above)
Comment (catalinaperalta) [reply src/go/cmd/root.go:17]: Done.

File: src/go/README.md (hunk 3 above)
Comment (catalinaperalta) [reply src/go/README.md]: Done

File: src/go/README.md (hunk 1 above)
Comment (catalinaperalta) [reply src/go/README.md]: Done

**Review by jhendrixMSFT — APPROVED** [review]
